lib/message/decrypt.js

```diff
@@ -12,7 +12,7 @@ export async function decryptMessage(options) {
 
     try {
         const result = await openpgp.decrypt(options);
-        const { data, filename, verified, signatures, errors } = await handleVerificationResult(
+        const { data, filename, verified, errors, signatures } = await handleVerificationResult(
```

Comment:
This becomes very nitpicky but... why? This looks stranger than what it was IMO. Also below (`errors, signatures: pgpSignatures`).

Reply:
true, it's Frankestein code